modules/yarn.lock

```diff
@@ -5842,6 +5853,11 @@ clay@2.18.1:
   resolved "https://registry.yarnpkg.com/clay/-/clay-2.18.1.tgz#c13bacd70d3e12b4da26b57d3506eb5046224f2d"
   integrity sha512-BDCpIpMjjgwHh3hnjt3UtIYaHvi41of5HmcL8KqMuMIy8KmzLM/7NDVphxU0G7IjisLS6RqTKL4gDlrss6xTyw==
 
+clay@^2.4.1:
+  version "2.21.4"
+  resolved "https://registry.yarnpkg.com/clay/-/clay-2.21.4.tgz#854a645ebe78eeaadd23660a060a0c5851639d44"
+  integrity sha512-k9rtzpBH2Dr0k0uiA20+EJQJl5CNoSjBruMlJQtWIhvSL4vmFzjz3fFL6yQD91OKCDdJYK7WkaTflbIX7TEDwQ==
+
```

Comment:
This one is a bit concerning (2.4.1 vs 2.18.1) ie. 15 Nov 2018 vs 10 Sep 2019 🙏 

The d3 dupes look relatively innocuous.

Reply:
Yeah, the thing is they're mostly `devDeps` which I have no idea how are passing SF... so I figured we'll take a nice time this week to sort out the whole commerce integration to check for anomalies like this.

Reply:
Probably these suppressions:

- https://github.com/liferay/liferay-portal/blob/06c02ecf4e21e01838db58ef8455a9710d0170d4/modules/apps/commerce/source-formatter-suppressions.xml#L83-L89
- https://github.com/liferay/liferay-portal/blob/06c02ecf4e21e01838db58ef8455a9710d0170d4/modules/dxp/apps/commerce/source-formatter-suppressions.xml#L10-L12